packages/ember/README.md

```diff
@@ -0,0 +1,73 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry SDK for Ember.js
+
+## Links
+
+- [Official SDK Docs](https://docs.sentry.io/quickstart/)
+- [TypeDoc](http://getsentry.github.io/sentry-javascript/)
+
+## General
+
+This package is an Ember addon that wraps `@sentry/browser`, with added functionality related to Ember. All methods available in
+`@sentry/browser` can be imported from `@sentry/ember`.
+
+### Installation
+
+As with other Ember addons, run:
+`ember install @sentry/ember`
+
+Then add the following config to `config/environment.js`
+
+```javascript
+  ENV['@sentry/ember'] = {
+    sentry: {
+      dsn: '__DSN__' // replace __DSN__ with your DSN
+    }
+  };
+```
+
+### Usage
+
+To use this SDK, call `SentryForEmber` before the application is initialized, in `app.js`. This will load Sentry config from `environment.js` for you.
+
+```javascript
+import Application from '@ember/application';
+import Resolver from 'ember-resolver';
+import loadInitializers from 'ember-load-initializers';
+import config from './config/environment';
+import { SentryForEmber } from '@sentry/ember';
+
+SentryForEmber();
```

Comment:
Can we make this `InitSentryForEmber()` to make it more explicit on what is going on?

Reply:
Nice, will do.